src/utils/api.js

```diff
@@ -189,6 +189,19 @@ export const fetchRaceResultsByCircuit = async (year, circuitId) => {
   }
 };
 
+export const fetchDriverColor = async (driver, session) => {
```

Comment:
tried to just copy from the above call.

was able to generate the correct url, but was getting cors error in console